src/raps/registerENS.ts

```diff
@@ -112,6 +123,74 @@ export const createSetNameENSRap = async (
   return newRap;
 };
 
+export const createTransferENSRap = async (
+  ensActionParameters: ENSActionParameters
+) => {
+  let actions: RapENSAction[] = [];
+
+  const {
+    clearRecords,
+    records,
+    setAddress,
+    transferControl,
+    toAddress,
+  } = ensActionParameters;
+
+  if (clearRecords) {
+    const emptyRecords = Object.keys(records || {}).reduce(
+      (records, recordKey) => ({
+        ...records,
+        // Use zero address for ETH record as an empty string throws an error
+        [recordKey]: recordKey === ENS_RECORDS.ETH ? AddressZero : '',
+      }),
+      {}
+    );
+
+    let newRecords: Records = emptyRecords;
+    if (setAddress && toAddress) {
+      newRecords = {
+        ...newRecords,
+        ETH: toAddress,
+      };
+    }
+
+    const ensRegistrationRecords = formatRecordsForTransaction(newRecords);
+    const validRecords =
+      !isEmpty(emptyRecords) &&
```

Comment:
i think we could remove isEmpty? just to not use lodash

Reply:
do we have a native version of `isEmpty`?

Reply:
if not, should we defer this change until the `isEmpty` lodash migration?